src/utils/validators.js

```diff
@@ -0,0 +1,325 @@
+import _isUndefined from 'lodash/isUndefined'
+import _isArray from 'lodash/isArray'
+import _isEmpty from 'lodash/isEmpty'
+import _isObject from 'lodash/isObject'
+
+import { BaseOperation } from '../base/operations/base_operation'
+import { Keypair } from '../base/keypair'
+import { default as xdr } from '../base/generated/xdr_generated'
+
+/**
+ * Validators for encapsulating types checks and throwing errors.
+ *
+ * @example
+ * import { validateUndefined } from './validators'
+ *
+ * function someFunction (someParam) {
+ *   validateUndefined({
+ *     value: someParam,
+ *     fieldName: 'Some field'
+ *   })
+ *
+ *   // do something with someParam
+ * }
+ */
+
+/** Common types validators */
+
+/**
+ * Validate value to be not undefined.
+ *
+ * @param {any} value The value to validate.
+ * @param {string} [fieldName] The name of the field, containing value.
+ *
+ * @throws {TypeError} Value should be defined.
+ */
+export function validateUndefined ({ value, fieldName = '' }) {
```

Comment:
from name it follows that `undefined` is a valid value here - `validateNotUndefined` at least

Reply:
Renamed